### ISSUE TITLE
reading tprs

### ISSUE THREAD

**Issue by jmbuhr** [graeter-group/kimmdy#477]
https://userguide.mdanalysis.org/stable/formats/reference/tpr.html#supported-versions

**Comment by ehhartmann** [comment]
maybe this works for hat-naive but not for the proper hat reaction plugin because the files contain different information and are parsed differently in mda

**Comment by jmbuhr** [comment]
true, so we need to use mda directly from github. We can pin it to a specific commit on their default branch for our own dev dependencies in `requirements.txt`. Note though, that this will not work for KIMMDY's actual dependencies in setup.cfg, because PyPi doesn't allow that (https://setuptools.pypa.io/en/latest/userguide/dependency_management.html#direct-url-dependencies)

**Comment by jmbuhr** [comment]
This also means the HAT plugin can not make its way onto pypi until mda has had a new release on pypi
